### PR TITLE
Fixed bug on exp time validation

### PR DESCRIPTION
**Pull request by dev-tim**
There was a silly bug on jwt exp time validation. Replaced with dummy token fro longer expiration time and fixed the typo.

### CODE REVIEW

File: app/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationService.scala [hunk 1]
```diff
@@ -88,7 +88,7 @@ class OAuth2TokenValidationService(oAuth2ProviderSettings: OAuth2ProviderSetting
       }
 
       res.exp.fold(ifEmpty)(time =>
-        if (time <= System.currentTimeMillis) {
+        if (time >= System.currentTimeMillis / 1000) {
```
Comment (mr-mig) [inline app/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationService.scala:91]: Don't we have JodaTime?

File: test/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationServiceTest.scala [hunk 2]
```diff
@@ -15,29 +16,34 @@ class OAuth2TokenValidationServiceTest extends BaseSpec with Matchers with Route
   val mockTokenValidationService = ValidationTestRegistry.tokenValidationService
   val realValidationService = ValidationServiceRegistry.tokenValidationService
 
-  "Validation service" should "reject dummy token after validation remote server" in {
+  "Validation service" must "reject dummy token after validation remote server" in {
     whenReady(mockTokenValidationService.validate(Some(wrongToken))) { either =>
-      either should be('left)
+      either must be('left)
```
Comment (mr-mig) [inline test/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationServiceTest.scala:21]: What is the point of having `must` vs `should`?

File: test/com/yetu/oauth2resource/utils/RoutesHelper.scala [hunk 3]
```diff
@@ -9,9 +9,9 @@ trait RoutesHelper {
...
+  val correctToken = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzUxMiJ9.eyJzY29wZSI6ImJhciIsImlhdCI6MTQyMjQ2MDAyMCwiZXhwIjoxNzMzNTAwMDIwLCJhdWQiOiJ0ZXN0IiwiaXNzIjoiaHR0cHM6Ly9hdXRoLnlldHVkZXYuY29tIiwic3ViIjoic3ViamVjdC53aXRoLmRvdHMifQ.NNMlDrVNV3iLJCzuIA9dJcArZE69evas-z1H5mgwHGhOCoZmItXkzrZm_VbCY6EqaMYseAyF6agQddYNvaUZEEc1rq9vgN3XpeCiT6oVktjY6D-yu70SfeG1VSTpicyQX-VmzKKxj2Qn2DHJ_qF-Ck7mCY5Ki_qngMWYkxiV2Hq3Mfycz9zbBOWhg-aGPhn129cNAX_J0sswG2BGriLbxfnrvQ6PBMvlfTsXrqeLQP2dIaw0vXciK1lx8JvCkHQGurCV719PXDD1CatdFdBMlaVN7o-qUjkpRI0RHwAv0ppQOlO14et75DvyDwbMEEVvZvno0UnqNAb1uEQbhlqnQgny3jin8i3q--CQeiQXQEtLa2lIlvQYYSQVNOGddRi5KRMMLD9_DkFacQw-yV3srnSfHZGC_hjKtcT5taDP-BBlQcQWVdetoQOYwYWJTLgOZDfNBZjeTKcwiMWSqJSTMFhMrt9E9V4uWTRhg1sMtGqS-ZqLqmfPE3xLujbpSRAD-4Q3NKOcLAkloitpsmwD1bzfic1AVGfDR6Pl01Hya2Pq2tIquo-Jp98vBURIEVMPAubIvWWhlfNiTK04FhMMEETAKrC6btVValmIWOfS16IdUD1_QUE12tGt2Nw6PCdJp2gwwvfyYbOVfNcwnpUGbrGhZLETc37Lcl8wGIxiORw"
```
Comment (mr-mig) [inline test/com/yetu/oauth2resource/utils/RoutesHelper.scala:12]: The symbol number 233 is incorrect :trollface:

File: test/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationServiceTest.scala (hunk 2 above)
Comment (jschaul) [inline test/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationServiceTest.scala:21]: 99% of all scala test code written across all our projects use "must"-style matching. We even once had a document about such and other conventions. It makes no difference other than making it easier for new developer to dive into different projects and know the syntax.

File: test/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationServiceTest.scala [hunk 4]
```diff
@@ -15,29 +16,34 @@ class OAuth2TokenValidationServiceTest extends BaseSpec with Matchers with Route
...
     }
   }
 
-
+  "Validation service" must "validate access token (jwt) and make sure that JWT is valid at least for next month" in {
+    whenReady(realValidationService.validate(Some(correctToken), JWTTokenMethod)) { either =>
+      either must be('right)
+      either.right.get.exp.get must be > System.currentTimeMillis() / 1000 + (60 * 24 * 30)
```
Comment (jschaul) [inline test/com/yetu/oauth2resource/services/tokenvalidation/OAuth2TokenValidationServiceTest.scala:46]:
```
either.right.get.exp.get must be > System.currentTimeMillis() / 1000 + (60 * 24 * 30)
```

is testing not for 30 days, but for half a day.
if you wish to test for 30 days, use `> System.currentTimeMillis() / 1000 + (3600 * 24 * 30)` or add joda time and use the proper constants so one doesn't need to spend half a minute to make sure each of these lines of constructing the value of "month" is actually correct.

**Pull request merged**